presentation/src/main/java/me/androidbox/presentation/agenda/screen/AgendaScreen.kt

```diff
@@ -92,6 +100,19 @@ fun AgendaScreen(
             agendaScreenEvent(AgendaScreenEvent.OnDateChanged(localDate))
         }
     )
+
+    AgendaDropDownMenu(
+        modifier = Modifier.background(color = MaterialTheme.colorScheme.dropDownMenuBackgroundColor),
+        shouldOpen = agendaScreenState.agendaDropdownItemId.isNotEmpty(),
+        onCloseDropdown = {
+            /* Empty the list of item ids so the isExpanded dropdown will close as the items are empty */
+            agendaScreenEvent(AgendaScreenEvent.OnShowDropdown(listOf()))
```

Comment:
When the user clicked on the background screen. Set the state to an empty list so that the dropdownmenu will dismiss. 
i.e. `shouldOpen = agendaScreenState.agendaDropdownItemId.isNotEmpty()`

Reply:
Why not simply using a Boolean instead of modifying the list?

Reply:
@MKiperszmid Updated to use boolean